### PR TITLE
shl CalculateContamination tag doc, flesh out example command, mentio…

### PR DESCRIPTION
**Pull request by sooheelee**
…n replaces ContEst

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/CalculateContamination.java [hunk 1]
```diff
@@ -18,20 +19,35 @@
 import java.util.stream.IntStream;
 
 /**
- * Given pileup data from {@link GetPileupSummaries} calculate the fraction of reads coming from exogenous human DNA.
+ * Given pileup data from {@link GetPileupSummaries}, calculates the fraction of reads coming from cross-sample contamination.
  *
- * This tool is run as follows:
+ * <p>
+ *     The resulting contamination table is used with either {@link org.broadinstitute.hellbender.tools.walkers.mutect.Mutect2}
```
Comment (davidbenjamin) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/CalculateContamination.java]: Just `FilterMutectCalls`, I believe.  You can check their associated `ArgumentCollection` classes to verify.

**Comment by davidbenjamin** [comment]
@sooheelee back to you.

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/CalculateContamination.java (hunk 1 above)
Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/CalculateContamination.java]: If I `gatk-launch Mutect2`, I see that Mutect2 also has 
```
--contamination_fraction_per_sample_file,-contaminationFile:File
                              Tab-separated File containing fraction of contamination in sequencing data (per sample) to
                              aggressively remove. Format should be "<SampleID><TAB><Contamination>" (Contamination is
                              double) per line; No header.  Default value: null. 
```

But I can remove this if you think that it is better to only filter contamination using CalculateContamination.

Comment (davidbenjamin) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/CalculateContamination.java]: @sooheelee Excellent point, however these are relics of shared HaplotypeCaller code.  Mutect2 doesn't do anything with these arguments.  This probably means I should move them out of shared code and into HaplotypeCaller-only code.

Comment (sooheelee) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/CalculateContamination.java]: That is good to know. I've removed mention of filtering with M2.

**Comment by sooheelee** [comment]
@davidbenjamin Let me know if the changes look good to you. Thank you.

**Comment by davidbenjamin** [comment]
@sooheelee I'm satisfied.

**Pull request merged**